e2e_workflow/view_github_report.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# View Playwright report stored in GitHub
+#   ./view_github_report.sh [github-repo-owner] [github-run-id]
+#  
+
+# Abort script on errors and unbound variables
+# https://bertvv.github.io/cheat-sheets/Bash.html
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+
+# Set variables
+SCRIPT_PATH="$(cd "$(dirname "${0}")"; echo $(pwd))"
+REPO_PATH="$(dirname ${SCRIPT_PATH})"
+REPO_NAME="$(basename ${REPO_PATH})"
+BLOB_DIR="${REPO_PATH}/blob-report/"
+REPORT_DIR="${REPO_PATH}/playwright-report/"
+
+# Get parameters from user
+if [ "$#" -lt 2 ]; then
+  echo "Usage: $0 [github-repo-owner] [github-run-id]"
+  echo "* github-repo-owner: GitHub repo owner"
+  echo "* github-run-id: GitHub Run ID of report to view"
+  exit
+fi
+REPO_OWNER="${1}"
+RUN_ID="${2}"
+
+# Delete previous report files
+for file in $(find "${BLOB_DIR}" -type f) $(find "${REPORT_DIR}" -type f); do 
+  [ -f "${file}" ] && echo "delete: ${file}" && rm "${file}"
+done
+
+# Get workflow artifact id and name
+ARTIFACT=$(gh api \
+  -H "Accept: application/vnd.github+json" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "/repos/${REPO_OWNER}/${REPO_NAME}/actions/runs/${RUN_ID}/artifacts" \
+  | jq '.artifacts[] | select(.name | contains("html-report")) | {id, name}')
```

Comment:
`jq` may not be installed on dev machines, might want to add that to instructions (unless maybe it's a dependency of `gh`?)

Reply:
Oh good call! I don't believe `jq` is a dependency of `gh`, so I'll make a note to install it in the README.